scripts/update-manpages

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# =============
+# = Functions =
+# =============
+
+cleanup() {
+        printf '👋  Change back to original directory “%s”\n' "$CURRENT_DIR"
+        cd "$CURRENT_DIR" || exit
+}
+
+# ========
+# = Init =
+# ========
+
+CURRENT_DIR="$PWD"
```

Comment:
Executing `pwd` sounds more reliable to me. (PWD can simply be modified.) Alternatively, we should fix the other scripts.